.github/actions/setup_env/action.yml

```diff
@@ -25,14 +27,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    # set up python that's used below to run convert-requirements-to-conda-yml.py
-    # after the environment is updated, 'python' will point to the
-    # conda-managed python and this python won't be used again
-    - name: Set up python
```

Comment:
I realized that the mac and windows runners also have python pre-installed, which should be sufficient to run `convert-requirements-to-conda-yml.py`. No need to set up python before creating the environment.